pkgs/servers/ldap/389/default.nix

```diff
@@ -53,6 +53,16 @@ stdenv.mkDerivation rec {
       url = "https://github.com/389ds/389-ds-base/commit/1fe029c495cc9f069c989cfbb09d449a078c56e2.patch";
       hash = "sha256-b0HSaDjuEUKERIXKg8np+lZDdZNmrCTAXybJzF+0hq0=";
     })
+    (fetchpatch {
+      name = "CVE-2024-2199.patch";
+      url = "https://git.rockylinux.org/staging/rpms/389-ds-base/-/raw/r9/SOURCES/0004-CVE-2024-2199.patch";
```

Comment:
Should the url refer to something more permanent than a branch name perhaps?

Reply:
It should, thanks for noticing. I did not copy/paste the correct URL. I will fix it once I get to a computer.